src/main/resources/templates/db_scripts/schema.json

```diff
@@ -167,6 +167,24 @@
           "tOps": "ADD",
           "caseSensitive": false,
           "removeAccents": true
+        },
+        {
+          "fieldName": "pickupServicePointId",
+          "tOps": "ADD",
+          "caseSensitive": false,
+          "removeAccents": false
+        },
+        {
+          "fieldName": "requesterId",
+          "tOps": "ADD",
+          "caseSensitive": false,
+          "removeAccents": false
+        }
+      ],
+      "fullTextIndex": [
+        {
+          "fieldName": "status",
```

Comment:
The "ginIndex" for status should be removed.
We probably need an b-tree "index", not a "fullTextIndex" for status, see https://issues.folio.org/browse/CIRCSTORE-215

Reply:
I'm happy to remove them, thanks! The GIN index is marked as `DELETE` and a b-tree index is added:
```
{
    "fieldName": "status",
    "tOps": "ADD",
    "caseSensitive": false,
    "removeAccents": false
}
```